src/content/blog/posts/iceberg-athena-dbt.md

```diff
@@ -33,20 +33,21 @@ The processing layer transforms the data through a series of batch procedures be
 
 Previously, we utilised [Glue PySpark](https://aws.amazon.com/blogs/big-data/dive-deep-into-aws-glue-4-0-for-apache-spark/) to standardise the data and employed Athena for data modelling. While Glue PySpark is effective for heavy-duty data processing, it requires specialised knowledge of distributed systems for efficient use. Rising costs and recurring out-of-memory failures with the Glue jobs prompted us to reevaluate our strategy. With the release of [Athena V3](https://aws.amazon.com/blogs/big-data/upgrade-to-athena-engine-version-3-to-increase-query-performance-and-access-more-analytics-features/), it made sense to explore migrating the standardisation step to Athena as well and unify the technology stack.
 
+### Athena for data processing (Updated Nov 2024)
 
-### Athena for scalability
+Amazon Athena is built on the open-source [Trino SQL Engine](https://trino.io/) and uses the [AWS Glue Data Catalog](https://docs.aws.amazon.com/glue/latest/dg/catalog-and-crawler.html), an [Apache Hive metastore](https://blog.jetbrains.com/big-data-tools/2022/07/01/why-we-need-hive-metastore/)-compatible catalogue, to manage metadata for data held in S3. This allows users to interact with structured data in S3 using SQL queries. Athena is serverless and operates within a [shared regional cluster](https://repost.aws/questions/QUdX6shGHrT-GDpuc_NDkSNA/how-does-athena-prepare-a-cluster-of-compute-nodes-for-a-specific-query), meaning all accounts in the same AWS region share the same pool of resources. It uses asynchronous processes and [quotas](https://docs.aws.amazon.com/athena/latest/ug/service-limits.html) to ensure effective and fair usage across accounts.
```

Comment:
Useful additional context - helps show the full tech stack driving transformation for those interested.